docs/index.rst

```diff
@@ -16,6 +16,8 @@ Contents:
    reference
    css-classes-reference
    style-guide
+   plugins
+   recipes
```

Comment:
Not sure if it's a good idea to separate recipes and plugins. I'd put them both on the same page unless there's some reason.

Reply:
Maybe.  `recipes` is intended to be "recipes for small plugins" and really "plugins" is intended to be the list of hooks of plugin API.  That page is already going to get plenty long over time I think... and I think having the API docs separate from examples of usage makes sense to me.

I think the title even says "Plugin API".

Reply:
I might rename the pages "plugin_api" and "recipes" does that make more sense to you?

Reply:
Maybe "plugin-api" and "plugin-recipes"?

Reply:
Sure.